distribution/lib/Standard/Visualization/0.0.0-dev/src/Helpers.enso

```diff
@@ -8,6 +8,7 @@ import Standard.Table.Data.Storage.Storage
 
 import project.Id.Id
 from project.Text import get_lazy_visualisation_text_window
+import project.Table as TableVis
```

Comment:
```suggestion
import project.Table as Table_Visualization
```

`CamelCase` is discouraged in Enso (in fact I think in the past it was supposed to be forbidden completely, i.e. every `C`apital letter would have to be preceded by `_`; after all it was allowed for exceptions like `AWS_Connector` or `SQLite_Dialect`, but still `FooBar` should rather be `Foo_Bar`)